elastic/datadog_checks/elastic/__about__.py

```diff
@@ -2,4 +2,4 @@
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
 
-__version__ = "1.22.1"
+__version__ = "1.22.2"
```

Comment:
You don't need to change the version for this. Once we approve your changes we will create a release and update version number.

Reply:
oh, i was trying to pass checks as much as possible.